settings/DevHome.Settings/Views/PreferencesPage.xaml

```diff
@@ -26,44 +27,23 @@
-                    <labs:SettingsExpander.HeaderIcon>
-                        <FontIcon Glyph="&#xE790;" />
-                    </labs:SettingsExpander.HeaderIcon>
-                    <labs:SettingsExpander.Items>
-                        <labs:SettingsCard HorizontalContentAlignment="Left" ContentAlignment="Left">
-                            <StackPanel>
-                                <RadioButton
-                                    x:Uid="Settings_Theme_Light"
-                                    Command="{x:Bind ViewModel.SwitchThemeCommand}"
-                                    GroupName="AppTheme"
-                                    IsChecked="{x:Bind ViewModel.ElementTheme, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter=Light, Mode=OneWay}">
-                                    <RadioButton.CommandParameter>
-                                        <xaml:ElementTheme>Light</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                                <RadioButton
-                                    x:Uid="Settings_Theme_Dark"
-                                    Command="{x:Bind ViewModel.SwitchThemeCommand}"
-                                    GroupName="AppTheme"
-                                    IsChecked="{x:Bind ViewModel.ElementTheme, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter=Dark, Mode=OneWay}">
-                                    <RadioButton.CommandParameter>
-                                        <xaml:ElementTheme>Dark</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                                <RadioButton
-                                    x:Uid="Settings_Theme_Default"
-                                    Command="{x:Bind ViewModel.SwitchThemeCommand}"
-                                    GroupName="AppTheme"
-                                    IsChecked="{x:Bind ViewModel.ElementTheme, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter=Default, Mode=OneWay}">
-                                    <RadioButton.CommandParameter>
-                                        <xaml:ElementTheme>Default</xaml:ElementTheme>
-                                    </RadioButton.CommandParameter>
-                                </RadioButton>
-                            </StackPanel>
-                        </labs:SettingsCard>
-                    </labs:SettingsExpander.Items>
-                </labs:SettingsExpander>
+                <labs:SettingsCard x:Uid="Settings_Theme">
+                    <labs:SettingsCard.HeaderIcon>
+                        <FontIcon Glyph="&#xE790;" />
+                    </labs:SettingsCard.HeaderIcon>
+                    <ComboBox x:Name="ThemeSelectionComboBox"

```

Comment:
NIT: I believe you can get around a custom converter by making a view model that houses the integer and string description and set the DataTemplate to the view model.

Reply:
I'm removing the converter and going with a code behind solution for selecting the current theme on page load.